erpnext/accounts/doctype/pos_session_invoices/pos_session_invoices.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
- Anything that is deleted goes in the table "Deleted Document". 
- Also having a numbering series is also a good way to identify missing invoices!

Depends on what is "good enough". Is there a definition for this?